department-of-veterans-affairs/caseflow
# Always display hearing time in edt/ro timezone for Video/formally-video hearings

Resolves #15132

### Description
- update `HearingTime` component to display hearing time in `edt/ro timezone` in daily docket for virtual and formally virtual hearings

### Testing Plan
1. Enable feature toggle on rails console`FeatureToggle.enable!(:full_page_video_to_virtual, users: [User.find_by(css_id: "BVASYELLOW")])`
2. Login as `BVASYELLOW`
3. Go to http://localhost:3000/hearings/schedule and pick a video hearing day
4. Convert the hearing day to `Virtual` 
5. Go to daily docket and notice that hearing time is displayed in `edt/ro timezone` in the `TYPE/TIME/RO` column
